include/uvgrtp/debug.hh

```diff
@@ -69,24 +69,24 @@ static inline void uvgrtp_debug(const char *level, const char *function, const c
 #endif
 
 #ifdef __RTP_SILENT__
-#define LOG_ERROR(...) ;
-#define LOG_WARN(...) ;
-#define LOG_INFO(...) ;
-#undef LOG_DEBUG
-#define LOG_DEBUG(...) ;
+#define UVG_LOG_ERROR(...) ;
+#define UVG_LOG_WARN(...) ;
+#define UVG_LOG_INFO(...) ;
+#define UVG_LOG_DEBUG(...) ;
 #else
-#define LOG_ERROR(...) uvgrtp_debug(LOG_LEVEL_ERROR, __func__, __VA_ARGS__)
-#define LOG_WARN(...)  uvgrtp_debug(LOG_LEVEL_WARN,  __func__, __VA_ARGS__)
-#define LOG_INFO(...)  uvgrtp_debug(LOG_LEVEL_INFO,  __func__, __VA_ARGS__)
+#define UVG_LOG_ERROR(...) uvgrtp_debug(LOG_LEVEL_ERROR, __func__, __VA_ARGS__)
+#define UVG_LOG_WARN(...)  uvgrtp_debug(LOG_LEVEL_WARN,  __func__, __VA_ARGS__)
+#define UVG_LOG_INFO(...)  uvgrtp_debug(LOG_LEVEL_INFO,  __func__, __VA_ARGS__)
+#define UVG_LOG_DEBUG(...)  uvgrtp_debug(LOG_LEVEL_DEBUG,  __func__, __VA_ARGS__)
```

Comment:
Both UVG_LOG_DEBUG should be modified into the LOG_DEBUG on line 66 and 68. This way, debug prints are disabled in release mode.